convert2rhel/unit_tests/grub_test.py

```diff
@@ -159,8 +159,15 @@ def test_get_device_number(monkeypatch, caplog, expected_res, device, exc, subpr
         assert len(caplog.records) == 0
 
 
-def test_get_device_number_no_output(monkeypatch):
-    monkeypatch.setattr("convert2rhel.utils.run_subprocess", RunSubprocessMocked(return_value=("", 0)))
+@pytest.mark.parametrize(
+    ("output"),
+    (
+        (""),
+        ("\n"),
+    ),
+)
+def test_get_device_number_no_output(monkeypatch, output):
+    monkeypatch.setattr("convert2rhel.utils.run_subprocess", RunSubprocessMocked(return_value=(output, 0)))
```

Comment:
This should be expanded to cover a few more edge-cases just to be sure

Reply:
Did a few more runs of blkid and only saw either empty string or \n

Reply:
As of right now it doesn't matter, but what if bugs are introduced or something happens where we get space for example? Since we know we'll get a string in a successful return we can at least rule out other types, but string could still be empty, newline, space

Reply:
Good point, i'll add space to the test cases